electrumsv/tests/test_wallet.py

```diff
@@ -151,7 +154,9 @@ def check_legacy_parent_of_standard_wallet(wallet: Wallet,
         # This account was created for the test and gets the correct flag
         assert account_keystore.get_masterkey_flags() == MasterKeyFlags.ELECTRUM_SEED
     else:
-        assert account_keystore.get_parent_keystore() is None
+        migration_number = wallet._storage.get('migration')
+        if migration_number < 27:
+            assert account_keystore.get_parent_keystore() is None
```

Comment:
I don't know why this is being checked. Do you know what a parent keystore means and why would they only be present for version 27?

Reply:
I wanted to check this with you because the database and migration history is complicated... But my impression was that prior to migration `27`, there was no such thing as a parent keystore. But starting from db migration 27 there is a parent keystore introduced.

So given this regtest wallet is migration `29`, it was failing at this step because `account_keystore.get_parent_keystore()` is **NOT** `None`